### PR TITLE
new-files

### PR DESCRIPTION
**Pull request by SaadMahi**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+.container a:hover:after{
+    inset: -3px;
+    filter: blur(10px);
+}
+
+.container a span{
+    position: absolute;
+    top:0;
+    left: 0;
```
Comment (murtazshaikh) [inline style.css]: grouping

**Review by murtazshaikh — APPROVED** [review]
LGTM!

File: style.css (hunk 1 above)
Comment (SaadMahi) [reply style.css]: LGTM means?

Comment (SaadMahi) [reply style.css]: how do i group them?
i want it on top then at extreme left.

Comment (murtazshaikh) [reply style.css]: Not that particular property, but in general I am saying. In whole stylesheet you have not done grouping.

Comment (murtazshaikh) [reply style.css]: > LGTM means?

Looks good to me

File: style.css [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+}
+
+.container a{
+    position: relative;
+    width: 160px;
+    height: 60px;
+    background: white;
+    display: inline-block;
+    margin: 20px;
+}
```
Comment (ManSOorcode) [inline style.css]:
```js

.container a{

    position: relative;

    width: 160px;
    height: 60px;
    margin: 20px;

    background: white;
    display: inline-block;
} 
```

grouping means group the similar style property into multiple group by spacing . we done this for providing clear understanding to other developer what we have done in styling. if your not understand well so you can check this link which can remove your doubt :
https://css-tricks.com/different-logical-ways-group-css-properties/

Comment (SaadMahi) [reply style.css]: All styles are now grouped, kindly verify.

Comment (SaadMahi) [reply style.css]: All styles are now grouped, kindly verify.

**Review by SaadMahi — COMMENTED** [review]
All similar styles have been grouped

**Review by ManSOorcode — APPROVED** [review]

**Pull request merged**